/**:
  ros__parameters:
    map_topic: "/map/vector_map"
    input_objects_topic: "/objects_in"
    output_objects_topic: "/objects_out"
    marker_topic: "/objects_exclude_on_route_marker"
    marker_on_ns: "objects_on_route"
    marker_off_ns: "objects_off_route"
    target_subtype: "no_detection_area"
    excluded_labels: [0, 1, 2, 3, 4]
      
      #const uint8 UNKNOWN = 0;
      #const uint8 CAR = 1;
      #const uint8 TRUCK = 2;
      #const uint8 BUS = 3;
      #const uint8 TRAILER = 4;
      #const uint8 MOTORCYCLE = 5;
      #const uint8 BICYCLE = 6;
      #const uint8 PEDESTRIAN = 7;
